### DISCUSSION TITLE
Pages build and deployment stuck, won't rerun for new PRs.

### DISCUSSION THREAD

**Discussion by GithubAnon0000** [community/community#147167] [Actions] [answered]
### Select Topic Area

Question

### Body

Hello!

I wanted to translate the anki manual and have it published as a github page. [My pages-build-deployment action](https://github.com/GithubAnon0000/githubanon0000.github.io/actions/workflows/pages/pages-build-deployment) is stuck at "queued" though, after I tried to re-run it. I cannot trigger a new action either (but can run actions for older commits, which doesn't help unfortunately).

I made sure that the settings are right, that actions have all neccessary permissions, I tried with another browser, rerunning old actions, pushing new commits and lots of other things that I could find on the internet (mostly githubs docs) and that the github ai bot thingy told me to check.

It's my first time attempting this so I have zero experience. All I want is to build the anki docs using mdbook and have it deployed in https://githubanon0000.github.io/. The action is stuck tough and my page 404s.

The branch that should be running on the page is "german_translation", the "gh-pages" branch had been successfully created.

As a sitenote: It seems that the "german_translation" branch had been deployed instead of the "gh-pages" branch, as I can navigate manually to the plain text version of "german_translation" branche's files (e.g. https://githubanon0000.github.io/src/intro.md).

Any help is greaty appreciated!

**Comment by fe-sean** [comment]
I have quite a bit of experience with Github Actions and am having the same issue. I have 8 repositories of which 2 are stuck in queued. Submitting new PRs does nothing. Canceling the job doesn't help either -- it just respawns. I'm not anywhere near any of my usage limits. They all have the same settings. I'm hoping someone from Github can help me!

The stuck repos:
https://github.com/finops-experts/logger
https://github.com/finops-experts/client_configs

If Github is out there PLEASE HELP!

**Comment by GithubAnon0000** [comment, marked as answer]
The action is still stuck for me but deployment and build now works (I just copied the mdbook.yml file from [here](https://github.com/open-spaced-repetition/anki-manual-zh-CN/tree/main)).

My issue thus had been solved.
